oem/ibm/libpldmresponder/file_io_type_lic.cpp

```diff
@@ -33,13 +33,17 @@ int LicenseHandler::updateBinFileAndLicObjs(const fs::path& newLicJsonFilePath)
     int rc = PLDM_SUCCESS;
     fs::path newLicFilePath(fs::path(licFilePath) / newLicenseFile);
     std::ifstream jsonFileNew(newLicJsonFilePath);
+    Json dataNew;
 
-    auto dataNew = Json::parse(jsonFileNew, nullptr, false);
-    if (dataNew.is_discarded())
+    try
+    {
+        dataNew = Json::parse(jsonFileNew);
```

Comment:
define dataNew here itself? auto dataNew

Reply:
We need to use this dataNew after try-catch, so can't use auto